package.json

```diff
@@ -48,6 +48,7 @@
     "ember-cli-string-utils": "^1.1.0",
     "ember-cli-test-info": "^1.0.0",
     "ember-cli-version-checker": "^2.1.2",
+    "ember-fetch": "^5.1.3",
```

Comment:
I think we should we make this a `peerDependency` instead to avoid trolling folks with multiple versions, and a `devDependency` for our own use.

Reply:
this still needs to be removed from here. this was probably my bad when I did partial work on this.